ci/filter-tasks.js

```diff
@@ -156,9 +156,7 @@ var getTasksToBuildForPR = function() {
         }
     });
     if (shouldBeBumped.length > 0) {
-        // TODO - change this to an error once its proven to be working.
-        console.log('##vso[task.logissue type=warning;sourcepath=ci/filter-task.js;linenumber=160;]The following tasks should have their versions bumped due to changes in common:', shouldBeBumped);
-        // throw new Error('The following tasks should have their versions bumped due to changes in common: ' + shouldBeBumped);
+        throw new Error('The following tasks should have their versions bumped due to changes in common: ' + shouldBeBumped);
```

Comment:
Did you confirm in logs that shouldBeBumped gets serialized correctly?

Reply:
Yep!